src/OrchardCore.Modules/OrchardCore.Alias/Startup.cs

```diff
@@ -72,6 +72,15 @@ public override void ConfigureServices(IServiceCollection services)
         services.AddScoped<IContentPartIndexHandler, AliasPartIndexHandler>();
         services.AddScoped<IContentTypePartDefinitionDisplayDriver, AliasPartSettingsDisplayDriver>();
 
-        services.AddScoped<IShapeTableProvider, AliasShapeTableProvider>();
+        services.AddScoped<IShapeTableProvider, ContentAliasShapeTableProvider>();
```

Comment:
I would register this just like you did for the Widget but only when OrchardCore.Content feature is enabled

Reply:
done